src/main/java/org/datadog/jenkins/plugins/datadog/DatadogGlobalConfiguration.java

```diff
@@ -1324,8 +1394,74 @@ public boolean isEmitConfigChangeEvents() {
      * @param emitConfigChangeEvents - The checkbox status (checked/unchecked)
      */
     @DataBoundSetter
+    @Deprecated
     public void setEmitConfigChangeEvents(boolean emitConfigChangeEvents) {
         this.emitConfigChangeEvents = emitConfigChangeEvents;
+        this.createIncludeLists();
+    }
+
+    /**
+     * Setter function for the included global configuration,
+     * accepting a comma-separated string of events.
+     *
+     * @param events - a comma-separated list of events to include for sending to agent.
+     */
+    @DataBoundSetter
+    public void setIncludeEvents(String events) throws InvalidAttributeValueException {
+        if (this.isOverlappingStrings(events, this.excludeEvents)) {
+            throw new InvalidAttributeValueException("Included events and excluded events contain an overlap.");
+        }  
+        
+        this.includeEvents = events;
+        this.includeEventsIsEnv = false;
+        this.createIncludeLists();
+    }
+
+    /**
+     * Getter function for the included global configuration, containing
+     * a comma-separated list of events to send to agent.
+     *
+     * @return a String array containing the events included global configuration.
+     */
+    public String getIncludeEvents() {
+        return includeEvents;
+    }
+
+    /**
+     * Setter function for the included global configuration,
+     * accepting a comma-separated string of events.
+     *
+     * @param events - a comma-separated list of events to exclude for sending to agent.
+     */
+    @DataBoundSetter
+    public void setExcludeEvents(String events) throws InvalidAttributeValueException{
+        if (this.isOverlappingStrings(events, this.includeEvents)) {
+            throw new InvalidAttributeValueException("Included events and excluded events contain an overlap.");
+        }
+        
+        this.excludeEvents = events;
+        this.excludeEventsIsEnv = false;
+        this.createIncludeLists();
+    }
+
+    /**
+     * Getter function for the included global configuration, containing
+     * a comma-separated list of events not to send to agent.
```

Comment:
```suggestion
     * a comma-separated list of events not to send.
```
nit since these can also be sent through the API